routes/tokenRoute.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
```

Comment:
this functionality is not needed in route so delete this file.

Reply:
this has been resolved.